pkg/cmd/server/start/start_master.go

```diff
@@ -454,12 +454,14 @@ func (m *Master) Start() error {
 			openshiftConfig.Informers.Start(utilwait.NeverStop)
 			openshiftConfig.Informers.StartCore(utilwait.NeverStop)
 			openshiftConfig.AuthorizationInformers.Start(utilwait.NeverStop)
+			openshiftConfig.TemplateInformers.Start(utilwait.NeverStop)
```

Comment:
You could run this via a goroutine, and add a line immediately after this one that waits for the caches to sync (blocking). WDYT @deads2k @sttts?